msg/AlarmInfo.msg

```diff
@@ -0,0 +1,25 @@
+# Alarm Number
+int32 number
+
+# Sub Code
+int32 sub_code
+
+# Alarm Name/Message
+string name
+
+# Contents
+string contents
+
+# Meaning
+string description
+
+# Cause, Remedy and Notes for Cause-Remedy pairs
+#motoros2_interfaces/msg/CauseRemedyPair[] cause_remedy
+#  # Cause
+#  string cause
+#
+#  # Remedy
+#  string remedy
+#
+#  # Notes
+#  string notes
```

Comment:
```suggestion
```

My suggestion would be to remove these fields. There's no easy way to get to this content (except using FTP, as discussed in https://github.com/Yaskawa-Global/motoros2/issues/14#issuecomment-1714031880, which will/may not work on all our supported controllers), and I'm wondering how useful it would be to include it.

Wouldn't users rely on the Yaskawa alarm documentation?

Reply:
> Wouldn't users rely on the Yaskawa alarm documentation

Most people don't know about the alarm manual. (I only learned about it just the other day from a comment that you made.)

And even if they do know that such a document exists, the average user is just going to email techsupport with a photo of the pendant. 

I think this data is quite valuable. Having said that... none of the MR2 alarms are going to be in that CSV unless we add them. This would add another step in the installation process that could go wrong. 

Short-term... I'd say we should remove this.